styles.css

```diff
@@ -0,0 +1,83 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0 24px;
+  width: 100vw;
```

Comment:
- [ ] I suggest you remove this `width: 100vh` fixed width as the content already takes up the space it needs. If you stretch it to the end of the screen, the menu icon won't appear correctly.